applications/tari_app_grpc/src/conversions/transaction_input.rs

```diff
@@ -63,6 +66,7 @@ impl TryFrom<grpc::TransactionInput> for TransactionInput {
                 PublicKey::from_bytes(input.sender_offset_public_key.as_bytes()).map_err(|err| format!("{:?}", err))?;
             let covenant = Covenant::from_bytes(&input.covenant).map_err(|err| err.to_string())?;
             let encrypted_value = EncryptedValue::from_bytes(&input.encrypted_value).map_err(|err| err.to_string())?;
+            let minimum_value_promise = MicroTari::zero();
```

Comment:
I would not default this here. 
I would add this to the GRPC field as well and then map it here